.gitignore

```diff
@@ -13,3 +13,5 @@
 
 # testing csv file
 /spec/testing.csv
+testdata/*.xml
```

Comment:
This file should go into spec/fixtures/files/ with the other sample xml (only if it adds something new that the other files don't have.)

Reply:
Moved to `spec/fixtures/files/sample-events.xml`